client/build.gradle

```diff
@@ -1,3 +1,12 @@
 dependencies {
     compile project(':common')
+    compile project(':server')
```

Comment:
Integration tests should be placed under `test` project. We can remove `test` project but in that case we would have to change the dependency from `compile` to `test`. It would be nicer if we had a separate integration test project.